ros/resources/v0/hosts.py

```diff
@@ -90,6 +91,8 @@ def get(self):
                 host['id'] = i.__dict__['id']
                 host['recommendation_count'] = 5
                 host['state'] = 'Undersized'
+                pro = PerformanceProfile()
+                pro.my_score
```

Comment:
You already have performance_profile instance (here it's `i`) then why can't you use it directly instead of calling `PeroformanceProfile`?
Example - `i.my_score`

Reply:
Also, instead `i` , will it be good to rename this variable to something else so that one can get what information is `stored` inside that variable